EnergyCorrelatorOptions.h

```diff
@@ -131,6 +135,8 @@ namespace EnergyCorrelatorOptions {
       .nBinsDr       = nBinsDr,
       .drBinRange    = binRangeDr,
       .ptJetBins     = ptJetBins,
+      .manualTwoPoint = manualTwoPoint,
+      .manualThreePoint = manualThreePoint,
```

Comment:
Similar to above. I also suggest naming the flag `doManualCalc` to make it consistent with the flag for the package calculation. See:
https://github.com/ruse-traveler/SEnergyCorrelator/blob/main/src/SEnergyCorrelatorConfig.h#L39

```suggestion
      .doManualCalc = doManualCalc,
```